DESCRIPTION

```diff
@@ -1,14 +1,14 @@
 Package: cancensus
 Type: Package
-Title: CensusMapper API Interface to Canada Census
+Title: Access to Canadian Census Data Through the 'CensusMapper' API
 Version: 0.1.0
 Authors@R: c(
     person("Jens", "von Bergmann", email = "jens@mountainmath.ca", role = c("aut", "cre")),
     person("Dmitry", "Shkolnik", email = "shkolnikd@gmail.com", role = "aut"),
-    person("Aaron","Jacobs", email = "atheriel@gmail.com", role = "aut"))
-Description: The package provides convenient access to StatCan census data via CensusMapper APIs.
-    An API key is required to access the data. API keys are available with free registration to CensusMapper at
-    https://censusmapper.ca/api
+    person("Aaron", "Jacobs", email = "atheriel@gmail.com", role = "aut"))
+Description: This package provides convenient and uniform access to to Canadian
```

Comment:
Should probably formally acknowledge Statcan here, and shapefiles refers to a specific type of geo format (and not one that we're using.)

"This package provides convenient and uniform access to to Statistics Canada census data and geography files through the..."

Reply:
I had though shapefile was more generic, but I see now that [this is false](https://en.wikipedia.org/wiki/Shapefile). I like using "boundary" because that seems to be the usual term for the census regions. How about "boundary geometry"?

It might be better to indicate that Statistics Canada is the original source of the data, because it's not inevitable that the Canadian census *must* be collected for all time by StatsCan, and I don't know how correct it is to equate the collection of the census with that agency. So how about

> "This package provides convenient and uniform access to Canadian census data and boundary geometry through the 'CensusMapper' interface to Statistics Canada's original data. API keys are available with free registration at..."

Reply:
>I had though shapefile was more generic, but I see now that this is false. I like using "boundary" because that seems to be the usual term for the census regions. How about "boundary geometry"?

Yeah shapefiles are a proprietary format. STC provides the data via shapefile, then censusmapper converts it into GeoJSON, which is then read as either SF or SP in cancensus. I have no problem with "boundary geometry".

> It might be better to indicate that Statistics Canada is the original source of the data, because it's not inevitable that the Canadian census must be collected for all time by StatsCan, and I don't know how correct it is to equate the collection of the census with that agency.

I think its fair to equate the two. The Census is Statistics Canada's mandate for existence under the [Statistics Act](http://laws.justice.gc.ca/eng/acts/S-19/FullText.html). As an aside, I only have this pedantic reference because I recently had to investigate the extent of the Statistics Act for a work thing 😆 )

Reply:
I did not know that!

Reply:
For a real trip, take a look at item 13 "Access to Records". Let's just say STC doesn't take advantage of its full legal powers.